iosApp/iosApp/ComponentViews/ErrorBanner.swift

```diff
@@ -46,6 +46,12 @@ struct ErrorBanner: View {
                 }
                 .frame(minHeight: minHeight)
             }
+        case .networkError:
+            ErrorCard { HStack {
```

Comment:
Is the default alignment in this HStack not center? It looks kinda like the image is below the centerline of the text.

Reply:
Added some borders to each view - I _think_ looks it is centered when accounting for the full line height of the text.
![image](https://github.com/user-attachments/assets/e486d814-07ab-4756-86fd-0365d152ac63)

I swapped in `HStack(alignment: .center)` and didn't notice a difference

Reply:
Okay yeah, I think it's just a matter of the icon being slightly bottom heavy. Thanks for checking!